### PR TITLE
Directional modal after dataset creation

### PR DESCRIPTION
**Pull request by RoriCremer**
Screen Shot 2019-04-06 at 2 35 21 PM

### CODE REVIEW

**Comment by ahaessly** [comment]
when i have a PR for a branch that is not off of develop, i usually create the PR with the base branch being the one it depends on so diffs aren't included in both PRs.
also, i like the 3rd example above.

File: src/components/DatasetCreateView.jsx
Comment (thathert) [inline src/components/DatasetCreateView.jsx]: `map` should only be used to create a new array. It feels strange to use it to mutate something like `assetOptions` here instead of taking a more functional approach. Either `forEach` should be used here, or we should switch assetOptions to use `let` above and then have this line be
```javascript
assetOptions = study.schema.assets.map(asset => ({ value: asset.name, label: asset.name }));
```
(I prefer using `map`)

File: src/components/DatasetCreateView.jsx
Comment (thathert) [inline src/components/DatasetCreateView.jsx]: Similar comment as below (let's switch to just assigning the result of the map to studyOptions)

File: src/components/DatasetDirectionalModal.jsx
Comment (thathert) [inline src/components/DatasetDirectionalModal.jsx]: nit: I know I suggested this text so feel free to throw something at me, but the use of next twice in this modal seems excessive now that I see it -- let's just have this be "Where to?"

**Review by thathert — APPROVED** [review]
Approving with the expectation that comments are addressed or discussed before merging

**Pull request merged**
